library/package.json

```diff
@@ -50,7 +50,9 @@
   },
   "dependencies": {
     "asyncapi-parser": "^0.8.2",
+    "constate": "^1.2.0",
     "dompurify": "^1.0.11",
+    "json-schema-ref-parser": "^7.1.1",
```

Comment:
this package is dependency of `asyncapi-parser`, is there really a need to install it explicitly? Afaik it works without installing it explicitly